__tests__/lib/cli/envAlias.ts

```diff
@@ -1,13 +1,20 @@
 /* eslint-disable jest/no-conditional-expect */
+/**
+ * External dependencies
+ */
+import { afterEach, describe, expect, it, jest } from '@jest/globals';
+
 /**
  * Internal dependencies
  */
-import envAlias from '../../../src/lib/cli/envAlias';
+import * as envAlias from '../../../src/lib/cli/envAlias';
```

Comment:
No default export so this needs to be `* as envAlias`.

Reply:
I think we have `esModuleInterop` set to `true`, which implies [`allowSyntheticDefaultImports`](https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports).

Reply:
That's what I thought so I tried `import envAlias` at first. I got an error instead. I think it only works with the `module.exports` system.